docs/language/values-and-types.mdx

```diff
@@ -1471,3 +1471,21 @@ and [`Equatable`](../interfaces#equatable-interface) [interfaces](../interfaces)
 
 Most of the built-in types, like booleans and integers,
 are hashable and equatable, so can be used as keys in dictionaries.
+
+## Paths
+
+Paths represent locations in account storage that objects can be stored and accessed at.
+
+They consist of a domain (`storage`, `private`, or `public`) and an identifier separated by forward slashes. e.g. `/storage/MyProjectCollectionResource` .
+
+### Path functions
+
+- `cadence•fun toString(): String`
+
+  Returns the string representation of the path.
+
+  ```cadence
+  let storagePath = /storage/path
+
+  storagePath.toString()  // is "/storage/path"
+  ```
```

Comment:
Nice. This should be moved to `accounts.mdx` under the `### Paths` section, probably as a level-4 heading (`#### Path Functions`)

Reply:
Done, thank you!